crates/orderbook/src/orderbook.rs

```diff
@@ -257,42 +252,37 @@ impl Orderbook {
         Metrics::on_order_operation(&old_order, OrderOperation::Cancelled);
         Metrics::on_order_operation(&new_order, OrderOperation::Created);
 
-        self.solvable_orders.request_update();
-
         Ok(new_order.metadata.uid)
     }
 
     pub async fn get_order(&self, uid: &OrderUid) -> Result<Option<Order>> {
-        let mut order = match self.database.single_order(uid).await? {
-            Some(order) => order,
-            None => return Ok(None),
-        };
-        set_available_balances(std::slice::from_mut(&mut order), &self.solvable_orders);
```

Comment:
Thanks for noticing that. I meant to write a comment about that but forgot.

Previously we were setting the available balance for all orders no matter how they were queried. We did this because we had the balance available anyway because we need it for creating the auction/solvable orders and it needs to be there because the solvers might need it.

For orders returned by the auction endpoint we still set the available balance. For orders returned by other endpoints we no longer set the available balance.